website/content/docs/connect/gateways/mesh-gateway/wan-federation-via-mesh-gateways.mdx

```diff
@@ -42,6 +40,13 @@ gateways](/docs/connect/gateways/mesh-gateway/service-to-service-traffic-datacen
 
 [![WAN federation with mesh gateways](/img/wan-federation-connectivity-mesh-gateways.png)](/img/wan-federation-connectivity-mesh-gateways.png)
 
+## Requirements
+
+- Consul 1.8.0+
+- Mesh federation between Consul datacenters: all datacenters must be connected to all other datacenters
+   - Other topologies, such as [hub-and-spoke enabled by Consul Enterprise network areas](https://learn.hashicorp.com/tutorials/consul/federation-network-areas),
```

Comment:
```suggestion
   - Other topologies, such as "hub-and-spoke" enabled by Consul Enterprise network areas, are not supported. Refer to the [Consul federation tutorial](https://learn.hashicorp.com/tutorials/consul/federation-network-areas) for a discussion about hub-and-spoke topologies.
```
I think it would be a good practice to start explicitly calling out when we are sending readers to content outside of Consul.io. It's just a small thing I think we can do to help orient readers when they are about to go to a different site.

Reply:
I would imagine it's generally true that it would be disorienting to be redirected to a different site. Do we have a sense of whether our users see our docs and Learn sites as "not one thing" (so the redirect is disorienting)? Or that they are seen as "one thing" that happens to be on two different domains with slightly different look and feel?

(This probably relates to longer-term thinking about the division of responsibilities between those two sites.)

Reply:
From what I understand, users generally view them as one thing, which is probably adds to the confusion when we link directly across the sites without some sort of heads up.

Reply:
In a different comment (on `federation.mdx`), you recommended:
> I think we should remove the link to information we're telling users not to implement.

Does that mean we should also not link to a hub-and-spoke guide here?